doc/src/tutorial/solvers.rst

```diff
@@ -179,7 +179,7 @@ multiplicity 1 and ``3`` is a root of multiplicity 2.
    ``solve`` can be used for such cases:
 
    >>> solve(x*exp(x) - 1, x )
-   [W(1)]
+   [W(1, integer)]
```

Comment:
The output from `solve` should not have this "integer" symbol in it.

Reply:
instead of integer can I replace it with `n` ?